doc/release-notes/6.5-release-notes.md

```diff
@@ -0,0 +1,367 @@
+# Dataverse 6.5
+
+Please note: To read these instructions in full, please go to https://github.com/IQSS/dataverse/releases/tag/v6.5 rather than the list of releases, which will cut them off.
+
+This release brings new features, enhancements, and bug fixes to Dataverse. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+Highlights in Dataverse 6.5 include:
+
+- new API endpoints, including editing of collections, Search API file counts, listing of exporters, comparing dataset versions, and auditing data files
+- UX improvements, especially Preview URLs
+- increased harvesting flexibility
+- performance gains
+- a security vulnerability addressed
```

Comment:
I don't think we ever put our a security announcement. We probably should. See https://guides.dataverse.org/en/6.4/developers/security.html#sending-security-notices